go/vt/tabletserver/planbuilder/dml.go

```diff
@@ -29,6 +29,11 @@ func analyzeUpdate(upd *sqlparser.Update, getTable TableGetter) (plan *ExecPlan,
 		return nil, err
 	}
 
+	// Store the WHERE clause as string for the transaction serializer.
+	buf := sqlparser.NewTrackedBuffer(nil)
+	buf.Myprintf("%v", upd.Where)
```

Comment:
nvm. Missed that part.
I was sitting at Starbucks, and typed this up for fun. It compiles, but no idea if it runs: https://github.com/sougou/vitess/commit/71ae9ff2d29e296fefa8bba55a3419589209c8cf.